myuw/templates/handlebars/card/summary/schedule.html

```diff
@@ -21,56 +21,44 @@ <h3 class="myuw-card-title">
                            data-linklabel="{{capitalizeString quarter}} {{year}} Teaching Details">View details</a>
                     </div>
                 {{/if}}
-                {{#each sections}}
-                    {{#unless under_disclosure}}
-                        <div class="myuw-card-section-fulldivider {{#if @last}}last{{/if}}">
-                            <div class="row">
-                            {{> summary_section_panel }}
-                            </div>
 
-                            {{#if linked_secondaries}}
-                                <div class="row"> <!-- one row for disclosure link and table-->
-                                    <div class="col-sm-12">
-                                        <div class="myuw-card-row-content">
-                                            <div class="card-disclosure">
-                                                <a class="collapsed myuw-disclosurelink" href="#secondary_{{section_label}}"
-                                                   role="button" aria-controls="secondary_{{section_label}}"
-                                                   aria-expanded="false" data-toggle="collapse" title="Show/Hide secondary sections"><i
-                                                   class="fa fa-caret-right" aria-hidden="true"></i>Secondary sections for
-                                                   {{ curriculum_abbr }} {{course_number}} {{section_id}}</a>
-                                            </div>
 
-                                            <div id="secondary_{{section_label}}" class="collapse secondary-section-disclosure"
-                                             rel="{{ curriculum_abbr }} {{course_number}} {{section_id}}"
-                                             aria-label="Secondary sections for {{ curriculum_abbr }} {{course_number}} {{section_id}}">
+             {{#if is_summer}}
+
+                {{#if full_term }}
+                 <div class="myuw-card-subheader">
```

Comment:
I think this should be `<h4 class="myuw-card-heading">` instead of a div. It is acting as a header so semantically that's the right tag. We can specifically adjust the top margin if it is too big.

Reply:
Great, I wasn't sure if using a div or header would be better. I was also curious if we needed to use the class - aren't all headers within a card going to look like this or have a separate class?